azblob/zt_highlevel_test.go

```diff
@@ -432,3 +432,31 @@ func (s *aztestsSuite) TestDoBatchTransferWithError(c *chk.C) {
 	mmf.isClosed = true
 	time.Sleep(time.Second * 5)
 }
+
+func (s *aztestsSuite) Test_CopyFromReader(c *chk.C) {
+	ctx := context.Background()
+	p, err := createSrcFile(_1MiB * 12)
```

Comment:
Just Curious: Why `12MiB`? If this any random number, can we keep the size a little less as it increases testsuite runtime on local

Reply:
So I was hoping to test a larger file, but we could potentially keep it smaller.